warp/yul/parse.py

```diff
@@ -75,7 +75,14 @@ def parse_literal(yul_ast) -> ast.Literal:
     elif kind == "bool":
         return ast.Literal(yul_ast["value"] == "true")
     elif kind == "string":
-        raise WarpException("string literals are not supported yet")
+        parsed_string = "".join([i if i.isascii() else "" for i in yul_ast["value"]])[
+            :32
```

Comment:
We don't need to do this anymore because YUL splits strings

Reply:
I'll keep the 32 just for sanity?

Reply:
I think it very much interferes with sane formatting =)

Reply:
fair

Reply:
Also, that `isascii` check isn't going to work at all AFAIU. Unicode symbols are not presented as-is but [as escapes](https://docs.soliditylang.org/en/latest/yul.html?highlight=optimization#literals)